python/py-setuptools/Portfile

```diff
@@ -26,8 +26,9 @@ homepage            https://pypi.org/project/setuptools/
 master_sites        pypi:s/setuptools/
 use_zip             yes
```

Comment:
Just curious, this has used the .zip distfile since 32.3.0 (4c7178e7f37d6c804cf338d466ee2b037c9ecfc7). I'm not aware of any differences, but would it be preferable to use the .tar.xz distfile instead? (The files used for the versions pinned for 2.6 and 3.3 would also be affected.)

Reply:
Where are you seeing a .tar.xz distfile?